### PR TITLE
fix radiant bug 'Mal-formed command line - please check for extra quotes in macro'

### PR DESCRIPTION
**Pull request by slagernate**
See #1596.

### CODE REVIEW

File: litex/build/lattice/radiant.py [hunk 1]
```diff
@@ -153,7 +153,9 @@ def tcl_path(path): return path.replace("\\", "/")
 
         # Add include paths
         vincpath = ";".join(map(lambda x: tcl_path(x), self.platform.verilog_include_paths))
-        tcl.append("prj_set_impl_opt {include path} {\"" + vincpath + "\"}")
+        if vincpath != "": # Do not add quotes if vincpath is empty, else Radiant complains
```
Comment (trabucayre) [inline litex/build/lattice/radiant.py:156]: An empty string is seen as `False` so:
````
if vincpath and vincpath.strip():
   [...]
```
is more pythoniic and catch empty string and string with only blank.

File: litex/build/lattice/radiant.py [hunk 2]
```diff
@@ -153,7 +153,9 @@ def tcl_path(path): return path.replace("\\", "/")
...
+            vincpath = "\"" + vincpath + "\"" 
+        tcl.append("prj_set_impl_opt {include path} {" + vincpath + "}")
```
Comment (trabucayre) [inline litex/build/lattice/radiant.py:158]: is `include path` is a mandatory option? ie when `vincpath` is empty this line may be skipped?

Comment (slagernate) [reply litex/build/lattice/radiant.py:158]: Seems to build fine without `include path`

Comment (slagernate) [reply litex/build/lattice/radiant.py:159]: @trabucayre see latest commit. Happy to make further changes / take feedback

Comment (trabucayre) [reply litex/build/lattice/radiant.py:159]: LGTM, but l.159 must be removed.

Thanks

Comment (slagernate) [reply litex/build/lattice/radiant.py]: Whoops. Thanks for spotting that.

**Comment by trabucayre** [comment]
Applied. Thanks @slagernate !

**Pull request merged**